src/Lykke.Sdk/AppLifetimeHandler.cs

```diff
@@ -67,6 +69,8 @@ public void HandleStarted()
                     .GetAwaiter()
                     .GetResult();
 
+                if (Debugger.IsAttached)
```

Comment:
Maybe it is better to use this flag here
https://github.com/LykkeCity/Lykke.Sdk/pull/27/files#diff-15255b81dea8652bf3b41f7de1283f8eR58 ?